src/vs/editor/contrib/codeAction/browser/codeActionModel.ts

```diff
@@ -165,6 +165,8 @@ export class CodeActionModel extends Disposable {
 	private readonly _onDidChangeState = this._register(new Emitter<CodeActionsState.State>());
 	public readonly onDidChangeState = this._onDidChangeState.event;
 
+	private readonly disposable: MutableDisposable<IDisposable> = this._register(new MutableDisposable());
```

Comment:
Could this be renamed to actionsDisposable and given the same type as codeActionSet and actionsAtMarker?